src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml

```diff
@@ -14,8 +14,8 @@ kuma_admin_list:
 
 # Filters
 filter:
-    true: true
-    false: false
+    "true": true
```

Comment:
shouldn't the value be quoted instead of the key?

Reply:
I think both the key and the value need to be quoted, so it seems when I'm testing..

Reply:
For me it worked if i only commented the key.

Reply:
but i will update everything to be sure